portfolio/src/main/webapp/script.js

```diff
@@ -75,3 +75,70 @@ function addQuoteToDom(quote) {
   const quoteContainer = document.getElementById('quote-container');
   quoteContainer.innerText = quote;
 }
+
+const google = window.google;
+window.onload = createMap;
+let map = window.map;
+
+function createMap() {
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: { lat: 42.8864, lng: -78.8784 },
+    zoom: 20,
+    mapTypeId: 'satellite',
+  });
+
+  const buffalo = new google.maps.Marker({
```

Comment:
Looks like you have four pieces of structured data that you use here (representing Buffalo, Howard, Carnegie Mellon, and NYC). Can you find a generic way to set up this logic such that you can add a fifth piece of data without changing any logic to `createMap`?

Reply:
Fixed, I split up the createMap function and added a createMarkers function